content_sync/pipelines/definitions/concourse/mass-build-sites.yml

```diff
@@ -118,7 +118,8 @@ jobs:
           npm run build:webpack
           npm run build:githash
           mkdir -p ./base-theme/static/static_shared/
-          cp -r ./base-theme/dist/static_shared/* ./base-theme/static/static_shared/
+          find ./base-theme/dist/static_shared/* ! -name '*.js.map' -type f | xargs cp -t ./base-theme/static/static_shared/
```

Comment:
So, I don't think this `find` / `xargs cp` strategy is going to work here. I tested this and ended up getting some errors:

```
cp: will not overwrite just-created './base-theme/static/static_shared/tex.js' with './base-theme/dist/static_shared/mathjax/output/svg/fonts/tex.js'
cp: will not overwrite just-created './base-theme/static/static_shared/tex.js' with './base-theme/dist/static_shared/mathjax/input/tex.js'
```

This is happening because this will only work with a flat directory structure. When we filter out the mp4 files, the `static_resources` folder is a flat structure so this works. One approach that I've considered in the past would be to add `rsync` to our `ocw-course-publisher` docker image. This would allow easy filtering with something like:

```
rsync -av --exclude='*.js.map' ./base-theme/dist/static_shared ./base-theme/static/static_shared
```

However, `rsync` is not part of our docker image. We could add it and revisit this, or alternatively for now we could do something like:

```
cp -r ./base-theme/dist/static_shared/* ./base-theme/static/static_shared/
find ./base-theme/static/static_shared -name "*.js.map" -type f -delete
```